test/terra/reference/ref_multiplexer.py

```diff
@@ -291,6 +291,12 @@ def multiplexer_no_control_qubits(final_measure=True):
     vector_circuit.isometry(vector / np.linalg.norm(vector), [0], None)
     vector_circuit = vector_circuit.inverse()
     qc.append(vector_circuit, [0])
+
+    sv = Statevector(qc)
+    gate_list = [np.array([[sv[0], -sv[1]], [sv[1], sv[0]]])]
+    qc = QuantumCircuit(1,1)
```

Comment:
I know we don't run lint in CI on tests (for whatever reason). But this would fail any format check:

```suggestion
    qc = QuantumCircuit(1, 1)
```